client-es6.json

```diff
@@ -2,6 +2,6 @@
 	"extends": [
 		"./client-common",
 		"./language/es6",
-		"./vue2-es6"
+		"./vue3-es6"
```

Comment:
Question from someone who's not very familiar with this repository: why not just extend `vue-es6` and specify the version in `vue-es6.json` only? I would think that would be a little bit more future proof since you'd only be defining the version in a single place.

Reply:
Hmm that's a good idea, I'll do that